src/contrib/loggers/Akka.Logger.SemanticSerilog/LogParser.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using Sprache;
+
+namespace Akka.Logger.SemanticSerilog
+{
+    internal static class LogParser
+    {
+        public static Parser<StringFormat> StringFormat =
```

Comment:
Removed sprache dependency, so maybe this could go in the standard serilog package as just an alternative logger class